alpaca_trade_api/rest.py

```diff
@@ -588,22 +598,19 @@ def get_trades_iter(self,
                 yield self.response_wrapper(trade, Trade)
 
     def get_trades(self,
-                   symbol: str,
-                   start: str,
-                   end: str,
+                   symbol: Union[str, List[str]],
+                   start: Optional[str] = None,
+                   end: Optional[str] = None,
                    limit: int = None,
                    ) -> TradesV2:
         trades = list(self.get_trades_iter(symbol,
-                                           start,
-                                           end,
-                                           limit,
-                                           raw=True))
+                                           start, end, limit, raw=True))
         return TradesV2(trades)
 
     def get_quotes_iter(self,
-                        symbol: str,
-                        start: str,
-                        end: str,
+                        symbol: Union[str, List[str]],
+                        start: Optional[str] = None,
```

Comment:
just a note, Optional is a Union of your type and None, so if you use Optional the default value is None

Reply:
This is exactly what I intended.